hbase-common/src/main/java/org/apache/hadoop/hbase/HConstants.java

```diff
@@ -156,6 +156,10 @@
   /** Default value for the balancer period */
   public static final int DEFAULT_HBASE_BALANCER_PERIOD = 300000;
 
+  /** Config for support separate child regions */
```

Comment:
I think this config needs more explaination about what it enables. Does it need a default setting too?

Reply:
Thanks, I'll add more explanation and a default value for it.